spec/controllers/fulfilments_controller_spec.rb

```diff
@@ -1,4 +1,4 @@
-require 'spec_helper'
+require 'rails_helper'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.